docs/docsite/rst/scenario_guides/guide_aci.rst

```diff
@@ -60,7 +60,7 @@ For instance ensuring that a specific tenant exists, is done using the following
         host: my-apic-1
         username: admin
         password: my-password
-    
+
```

Comment:
This was done on purpose, the block example continues.
It's one single example, not two paragraphs.
The only reason to not remove that line is because in these examples we want to separate the connection-part from the actual module parameters.

Reply:
hm, not sure where that change came from - I didn't mean to change anything in the guides themselves except for the titles

Reply:
No worries, most likely your editor is removing trailing spaces, which is mostly a good thing ;-)